wrims-gui/build.gradle

```diff
@@ -0,0 +1,143 @@
+        }
+    }
+}
+
+// Remove the dropins folder
+tasks.register('removeDropinsFolder', Delete) {
+    dependsOn renameFirstSubdirectory
+    delete "${buildDir}/installer/${wrimsGuiFolder}/dropins"
+}
+
+// Task that clears the workspace cache
+tasks.register('removeWorkspaceCache', Delete) {
+    dependsOn removeDropinsFolder
+    delete "${buildDir}/installer/${wrimsGuiFolder}/workspace/.metadata"
+}
+
+// Task to copy JARs to dropins folder
+tasks.register('copyJarsToDropins', Copy) {
+    dependsOn removeWorkspaceCache
+    from(project(':third-party').tasks.named('jar').get().archiveFile)
+    from(project(':wrims-ide').tasks.named('jar').get().archiveFile)
+    from(project(':dwr-hecdssvue').tasks.named('jar').get().archiveFile)
+    from(project(':jdiagram').tasks.named('jar').get().archiveFile)
+    from(project(':xtext-editor').tasks.named('jar').get().archiveFile)
+    from(project(':xtext-editor-ui').tasks.named('jar').get().archiveFile)
+    into("${buildDir}/installer/${wrimsGuiFolder}/dropins")
+}
+
+//task to delete all contents of the ${buildDir}/installer/${wrimsGuiFolder}/libs folder
+tasks.register('cleanLibFolder', Delete) {
+    dependsOn copyJarsToDropins
+    delete "${buildDir}/installer/${wrimsGuiFolder}/lib"
+}
+
+//task to download native libraries (dlls)
+tasks.register('getNatives', Sync) { syncTask ->
+    dependsOn cleanLibFolder
+    syncTask.from configurations.windows_x64.collect { zipTree(it) }
+    syncTask.into file("${buildDir}/installer/${wrimsGuiFolder}/lib")
+}
+
+// Task to copy JARs to dropins folder
+tasks.register('copyWrimsCoreToLib', Copy) {
+    dependsOn getNatives
+    from(project(':wrims-core').tasks.named('jar').get().archiveFile)
+    into("${buildDir}/installer/${wrimsGuiFolder}/lib")
+}
+
+//task to unzip the contents of the third-party jar into a temp folder
+tasks.register('unzipThirdPartyJar', Copy) {
+    dependsOn copyWrimsCoreToLib
+    from(zipTree(project(':third-party').tasks.named('jar').get().archiveFile))
+    into("${buildDir}/third-party")
+}
+
+//task to copy all files from ${buildDir}/third-party/lib_x64 to ${buildDir}/installer/${wrimsGuiFolder}/lib
+tasks.register('copyThirdPartyToLib', Copy) {
+    dependsOn unzipThirdPartyJar
+    from("${buildDir}/third-party/lib_x64")
+    from("${buildDir}/third-party/heclib")
+    into("${buildDir}/installer/${wrimsGuiFolder}/lib")
+}
+
+//task that copies the src/main/configuration/simpleconfigurator/bundles.info file into
+//bundles.info file contains the file mappings for the dropins jars.
+tasks.register('copyBundlesInfo', Copy) {
+    dependsOn copyThirdPartyToLib
+    from("src/main/configuration/simpleconfigurator/bundles.info")
+    into("${buildDir}/installer/${wrimsGuiFolder}/configuration/org.eclipse.equinox.simpleconfigurator")
+    filter { line ->
+        line.replace('${WRIMS_VERSION}', version)
+    }
+}
+
+//task that runs the bat file in build/download/${WRIMS_GUI_FOLDER/WRIMS2_GUI_Start.bat
```

Comment:
Comment needs to explain why this is being run

Reply:
Added this comment:

/**
 * Running the WRIMS2_GUI_Start.bat file will launch the WRIMS exe with a -clean argument that
 * forces an update of some ecplise files and ensures the user can then run the WRIMS GUI
 * with the updated packages using the WRIMS2GUI_x64.exe file going forward.
 */